addon/appModules/notepad++/editWindow.py

```diff
@@ -143,10 +150,29 @@ def script_reportFindResult(self, gesture):
 			#Translators: Message shown when there are no more search results in this direction using the notepad++ find command.
 			speech.speakMessage(_("No more search results in this direction."))
 	
-	#Translators: when pressed, goes to    the Next search result in Notepad++
+	#Translators: when pressed, goes to	   the Next search result in Notepad++
 	script_reportFindResult.__doc__ = _("Queries the next or previous search result and speaks the selection and current line of it.")
 	script_reportFindResult.category = "Notepad++"
 
+	def script_htmlPreview(self, gesture):
+		from threading import Timer
+		repeatCount=scriptHandler.getLastScriptRepeatCount()
+		raw=self.windowText
+		html = markdown.markdown(unicode(raw), extensions=['extra','toc', 'nl2br'])
+		if repeatCount==1:
+			import tempfile
+			f= tempfile.NamedTemporaryFile(suffix='.html',delete=False)
+			f.write(html)
+			f.close()
+			os.startfile(f.name)
```

Comment:
There's a webbrowser python module, specifically meant for opening web browsers. It might be better to look at using, in case the user for some reason opts for html to open in notepad, but I don't think I'd bother changing it at this point. More of an FYI for you in the future.